network-config-analyzer/nca.py

```diff
@@ -171,24 +197,29 @@ def nca_main(argv=None):
     manual_or_automatic = parser.add_mutually_exclusive_group(required=False)
     manual_or_automatic.add_argument('--scheme', '-s', type=_valid_path,
                                      help='A YAML scheme file, describing verification goals')
-    manual_or_automatic.add_argument('--sanity', type=_network_policies_valid_path,
+    manual_or_automatic.add_argument('--sanity', nargs='?', const='', type=_network_policies_valid_path,
                                      help='Network policies (file/dir/GHE url/cluster-type) for sanity checking')
-    manual_or_automatic.add_argument('--connectivity', type=_network_policies_valid_path,
+    manual_or_automatic.add_argument('--connectivity', type=_network_policies_valid_path, nargs='?', const='',
                                      help='Network policies (file/dir/GHE url/cluster-type) for connectivity map')
-    manual_or_automatic.add_argument('--semantic_diff', type=_network_policies_valid_path,
+    manual_or_automatic.add_argument('--semantic_diff', type=_network_policies_valid_path, nargs='?', const='',
                                      help='Network policies (file/dir/GHE url/cluster-type) for semantic-diff')
-    manual_or_automatic.add_argument('--equiv', type=_network_policies_valid_path,
+    manual_or_automatic.add_argument('--equiv', type=_network_policies_valid_path, nargs='?', const='',
                                      help='Network policies (file/dir/GHE url/cluster-type) for equivalence checking')
-    manual_or_automatic.add_argument('--interferes', type=str, help='Network policies '
-                                     '(policy name/file/dir/GHE url/cluster-type) for interference checking')
-    manual_or_automatic.add_argument('--forbids', type=str, help='Network policies '
-                                     '(policy name/file/dir/GHE url/cluster-type) specifying forbidden connections')
-    manual_or_automatic.add_argument('--permits', type=str, help='Network policies '
-                                     '(policy name/file/dir/GHE url/cluster-type) specifying permitted connections')
+    manual_or_automatic.add_argument('--interferes', type=str, nargs='?', const='',
```

Comment:
the types for interferes, forbids, permits, should also be network_policies_valid_path instead of str.